content/applications/inventory_and_mrp/inventory/product_management/product_tracking/lots.rst

```diff
@@ -1,197 +1,158 @@
 
-Once on the product form, click :guilabel:`Edit` to make changes to the form. Then, click the
-:guilabel:`Inventory` tab. In the :guilabel:`Traceability` section, click :guilabel:`By Lots`. Then,
-click :guilabel:`Save` to save changes. Existing or new lot numbers can now be assigned to
-newly-received or manufactured batches of this product.
+Once on the product form, click the :guilabel:`Inventory` tab. In the :guilabel:`Traceability`
+section, click :guilabel:`By Lots`. Then, click :guilabel:`Save`. Existing or new lot numbers can
+now be assigned to newly-received or manufactured batches of this product.
 
 .. important::
    If a product has stock on-hand prior to activating tracking by lots or serial numbers, an
    inventory adjustment might need to be performed to assign lot numbers to the existing stock.
 
-.. image:: lots/lots-tracking-product-form.png
+.. image:: lots/tracking-product-form.png
    :align: center
    :alt: Enabled tracking by lots feature on product form.
 
-Create new lots for products already in stock
----------------------------------------------
+Assign lots for shipping and receiving
+======================================
 
-New lots can be created for products already in stock with no assigned lot number. To do this, go to
-the :menuselection:`Inventory app --> Products --> Lots/Serial Numbers`, and click
-:guilabel:`Create`. Doing so reveals a separate page where a new :guilabel:`Lot/Serial Number` is
-generated automatically.
+New lot numbers can be assigned to incoming goods, on the receipt form. When shipping outgoing
+goods, choose products with specific lot numbers on the delivery order form.
 
-.. tip::
-   While Odoo automatically generates a new :guilabel:`Lot/Serial Number` to follow the most recent
-   number, it can be edited and changed to any desired number, by clicking the line under the
-   :guilabel:`Lot/Serial Number` field, and changing the generated number.
+.. _inventory/product_management/assign-lots:
 
-Once the new :guilabel:`Lot/Serial Number` is generated, click the blank field next to
-:guilabel:`Product` to reveal a drop-down menu. From this menu, select the product to which this new
-number will be assigned.
+On receipts
+-----------
 
-This form also provides the option to adjust the :guilabel:`Quantity`, assign a unique
-:guilabel:`Internal Reference` number (for traceability purposes), and assign this specific lot or
-serial number configuration to a specific website in the :guilabel:`Website` field (if working in a
-multi-website environment).
+Assigning new or existing lot numbers to incoming goods can be done directly on receipts.
 
-A detailed description of this specific lot or serial number can also be added in the
-:guilabel:`Description` tab below.
+To begin, go to the :menuselection:`Purchase` app to create and confirm a |PO| for products tracked
+by lot numbers. Then, select the :guilabel:`Receipt` smart button that appears at the top of the
```

Comment:
```suggestion
by lot numbers. Then, click the :guilabel:`Receipt` smart button that appears at the top of the
```
(This is such a small nit-pick, but I think we should reserve "select" for times when you are choosing items from a list or drop-down, and "click" when you are interacting with buttons or other UI elements)